### PR TITLE
Eran/mgmt 4111

### PR DESCRIPTION
**Pull request by eranco74**

### CODE REVIEW

File: internal/bminventory/inventory_test.go [hunk 1]
```diff
@@ -2077,6 +2077,31 @@ var _ = Describe("cluster", func() {
...
+					HighAvailabilityMode: &noneHaMode,
+				},
+			})
+			Expect(reflect.TypeOf(reply)).Should(Equal(reflect.TypeOf(installer.NewRegisterClusterCreated())))
+			actual := reply.(*installer.RegisterClusterCreated)
+			Expect(actual.Payload.HighAvailabilityMode).To(Equal(swag.String(noneHaMode)))
+			Expect(actual.Payload.UserManagedNetworking).To(Equal(swag.Bool(true)))
+		})
+
+		It("create non ha cluster fail", func() {
```
Comment (omertuc) [inline internal/bminventory/inventory_test.go:2101]: Can this test be more explicit about using a lower version number?

File: internal/bminventory/inventory.go [hunk 2]
```diff
@@ -586,6 +589,15 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 	}
 
 	if swag.StringValue(params.NewClusterParams.HighAvailabilityMode) == models.ClusterHighAvailabilityModeNone {
+		// verify minimal OCP version
+		ocp_version, err := version.NewVersion(swag.StringValue(params.NewClusterParams.OpenshiftVersion))
+		minimal_version_for_sno, err := version.NewVersion(minimalOpenShiftVersionForSingleNode)
+		if ocp_version.LessThan(minimal_version_for_sno) {
+			err := errors.Errorf("Invalid OCP version (%s) for HighAvailabilityMode=%s",
```
Comment (gamli75) [inline internal/bminventory/inventory.go:596]: I think we should change the message to: "Invalid OCP version for Single node, it's supported for version 4.8 & up".
Not sure that the user knows what it means HighAvailabilityMode=None
@avishayt WDYT?

File: internal/bminventory/inventory_test.go (hunk 1 above)
Comment (eranco74) [reply internal/bminventory/inventory_test.go:2101]: :+1:

**Pull request merged**